new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -18,6 +18,7 @@
   public static final String ADMIN_SERVER_NAME_BASE = "admin-server";
   public static final String MANAGED_SERVER_NAME_BASE = "managed-server";
   public static final String WLS_DOMAIN_TYPE = "WLS";
+  public static final String DEFAULT_WLS_IMAGE_TAGS = "12.2.1.3, 12.2.1.4";
```

Comment:
Not 14.1.1.0? I think we should add that version.

Reply:
modified